include/evmc/loader.h

```diff
@@ -0,0 +1,24 @@
+/* EVMC: Ethereum Client-VM Connector API.
+ * Copyright 2018 Pawel Bylica.
+ * Licensed under the MIT License. See the LICENSE file.
+ */
+
+#pragma once
+
+#if __cplusplus
+extern "C" {
+#endif
+
+enum evmc_loader_error_code
+{
+    EVMC_ERRC_SUCCESS = 0,
```

Comment:
Why not `EVMC_LOADER_SUCCESS`?

Reply:
@chfast this is still outstanding.

Reply:
At first I wanted to use a subset of  error codes from C++ errc: https://en.cppreference.com/w/cpp/error/errc, but in the end they didn't match.
We can either make it evmc-generic: evmc_loader_error_code -> evmc_error_code, or use `EVMC_LOADER_` prefix as you proposed.

Reply:
I'd vote for `EVMC_LOADER_`